client/blocks/reader-post-options-menu/README.md

```diff
@@ -19,18 +19,16 @@ function MyMenu() {
 
 ### `post`
 
-<table>
-	<tr><th>Type</th><td>Object</td></tr>
-	<tr><th>Required</th><td>Yes</td></tr>
-</table>
+| Type | Required |
```

Comment:
Kinda bugging me that the tables don't line up in the raw Markdown; this might look better:

```markdown
| Type   | Required |
| ------ | -------- |
| object | yes      |
```